taskchampion.pyi

```diff
@@ -59,7 +59,11 @@ class Operation:
     value: Optional[str]
 
 class Operations:
-    def append(op: "Operation"): ...
+    def append(self, op: "Operation"): ...
+    def __len__(self) -> int: ...
+    def __iter__(self) -> Iterator["Operation"]: ...
+    def __next__(self) -> "Operation": ...
```

Comment:
I think this is implemented by the iterator, but not by the collection, so it shouldn't be included here